app/services/notifications/v6m0/MessageDetails.scala

```diff
@@ -0,0 +1,204 @@
+      case ReminderToPayForRenewal => reminderPayRenewal(referenceNumber)
+      case ReminderToPayForApplication => s"""<p class="govuk-body">You need to pay $paymentAmount for your application to register with HM Revenue and Customs.</p><p class="govuk-body">Your payment reference is: $referenceNumber.</p><p class="govuk-body">Find details of how to pay on your status page.</p><p class="govuk-body">It can take time for some payments to clear, so if you’ve already paid you can ignore this message.</p>"""
+      case ReminderToPayForManualCharges => s"""<p class="govuk-body">You need to pay $paymentAmount for the recent charge added to your account.</p><p class="govuk-body">Your payment reference is: $referenceNumber.</p><p class="govuk-body">Find details of how to pay on your status page.</p><p class="govuk-body">It can take time for some payments to clear, so if you’ve already paid you can ignore this message.</p>"""
+      case _ => throw new Exception("An Unknown Exception has occurred, v6m0:reminder():MessageDetails")
+    }
+  }
+
+  val amlsRegistrationFeesGuidanceUrl = "https://www.gov.uk/guidance/money-laundering-regulations-registration-fees"
+  val emailUrl = "mailto:MLRCIT@hmrc.gov.uk"
+
+  def newRenewalReminderMsg(endDate: String): String =
+    s"""
+       |<p class="govuk-body">This is a reminder that you need to submit your annual declaration and pay your annual supervision fee. Please note that this is
+       |a <strong>two stage process</strong> and both stages <strong>must</strong> be competed to ensure we can process your fee.</p>
+       |
+       |<p class="govuk-body"><h3 class="govuk-heading-s">Stage 1 - Submit your declaration all your details and information  are correct and up to date.
+       |You must press the submit button to make that declaration which completes this stage.</h3></p>
+       |
+       |<p class="govuk-body">You start this stage from your status page on your anti-money laundering supervision account. You must check that all your details
+       |and information are up to date and make any necessary changes to the details information you have previously provided.</p>
+       |
+       |<p class="govuk-body"><strong>Please note -</strong> Even if your previous details and information have not changed, you must press the submit button to complete stage 1
+       |in order for us to be able to process your fee payment.</p>
+       |
+       |<p class="govuk-body"><h3 class="govuk-heading-s">Stage 2 - Pay your fee<h3><p>
+       |<p class="govuk-body">Once you have completed stage 1, the system will inform you of your new annual fee and generate a payment reference number which
+       |you must quote when you pay your fee.</p>
+       |
+       |<p class="govuk-body">HMRCs money laundering registration policy, and details of how to pay your fee can be found on
+       |<a href=$amlsRegistrationFeesGuidanceUrl">Fees you’ll pay for money laundering supervision - GOV.UK (www.gov.uk)</a></p>
```

Comment:
This link isn't working when I clicked it in the browser - the URL it sends me to has a quotation mark at the end